packages/venia-ui/lib/components/WishlistPage/__tests__/createWishlist.ee.spec.js

```diff
@@ -4,8 +4,22 @@ import { createTestInstance } from '@magento/peregrine';
 import CreateWishlist from '../createWishlist.ee';
 
 jest.mock('../../../classify');
+jest.mock('../../Dialog', () => props => (
+    <div componentName={'Dialog'} {...props} />
+));
+jest.mock(
+    '@magento/peregrine/lib/talons/WishlistPage/useCreateWishlist',
+    () => ({
+        useCreateWishlist: jest.fn().mockReturnValue({
+            createList: jest.fn(),
```

Comment:
I've found it beneficial to name these mock functions, easier to spot breakages in the snaps.

```suggestion
            createList: jest.fn().mockName('createList'),
```